### DISCUSSION TITLE
Flow diagram for argocd notifications

### DISCUSSION THREAD

**Discussion by brunocascio** [argoproj/argo-cd#11577] [Ideas]
Is there any Flow diagram explaining how argocd evaluates the argocd-notifications config map?

How triggers and defaultTriggers are processed along with subscriptions and defaultSubscriptions?

Thanks in advance

**Comment by pasha-codefresh** [comment]
Hey, i have not seen such diagram, i think we need open github issue for that

**Reply by brunocascio** [reply]
Thank you, I've just created it https://github.com/argoproj/argo-cd/issues/11648
